contributors.md

```diff
@@ -1,49 +1,58 @@
 # Contributing
 
 Dataform is a TypeScript project, and is fairly easy to build and run locally.
-For help making changes, join our `#development` channel in [Slack](https://slack.dataform.co).
+For help making changes, join our `#development` channel in [Slack](https://slack.dataform.co), or [email us](mailto:opensource@dataform.co) directly.
```

Comment:
is this actually an email address we have? or will this just all go to guillaume?